overview_report.py

```diff
@@ -0,0 +1,249 @@
+import argparse
+import csv
+from dataclasses import dataclass
+import logging
+from pathlib import Path
+import sys
+from typing import (
+    Mapping,
+    Sequence,
+)
+from uuid import UUID
+
+from count_cells import (
+    get_accession_ids,
+    get_cell_counts,
+)
+from create_project import (
+    generate_project_uuid,
+    get_spreadsheet_paths,
+)
+
+logging.basicConfig(level=logging.INFO)
+
+projects_path = Path('projects')
+
+
+@dataclass
+class ProjectReport:
+    uuid: UUID = None
+    accession_id: str = None
+    project_path: Path = None  # projects/{uuid}
+    accession_symlink: Path = None  # projects/{accession} symlink to project_path
+    spreadsheet: Path = None  # spreadsheets/(new|existing)/{accession}.0.xlsx
+    geo_files_count: int = 0  # number of downloaded geo files in projects/{uuid}/geo
+    matrices_count: int = 0  # number of matrices in projects/{uuid}/matrices
+    zipped_matrix: Path = None  # projects/{uuid}/bundle/matrix.mtx.zip
+    cell_count: int = 0  # value of cell count in cell_counts.json
+    metadata_json_count: int = 0  # number of metadata JSON files in projects/{uuid}/bundle
+
+    @classmethod
+    def tsv_header(cls) -> Sequence[str]:
+        return [
+            'UUID',
+            'Accession ID',
```

Comment:
```suggestion
            'accession',
```

Accessions *are* identifiers.